packages/bpk-tokens/tokens/base.raw.json

```diff
@@ -893,16 +893,16 @@
     },
     "BUTTON_ICON_BORDER_RADIUS": {
       "category": "buttons",
-      "value": "1.125rem",
+      "value": "50%",
       "type": "size",
-      "originalValue": "{!BORDER_RADIUS_PILL}",
+      "originalValue": "50%",
```

Comment:
Just one question about this, shouldn't we be updating and using the same token as before (BORDER_RADIUS_PILL) instead of `50%` directly? 

What happens if people are using `BORDER_RADIUS_PILL` directly in this case?

Reply:
As the `BORDER_RADIUS_PILL` token is used by other components it could have unintended sideaffects to those components that need it vs the button. If the other dependent components also need this set to 50% (covered in other tickets) then we can update the token at the route to keep it consistent

Reply:
Right, looks go to me then. We should try to revisit this in the future though

Reply:
Have made a note to revisit as the project goes along!